server/game/game.js

```diff
@@ -264,6 +264,19 @@ class Game extends EventEmitter {
     }
 
     isDuringConflict(types) {
+        if(!this.currentConflict) {
+            return false;
+        } else if(this.currentConflict && !this.currentConflict.conflictStarted) {
```

Comment:
I don't think this is a good idea - the engine shouldn't need to track whether the conflict has actually started or not here.